### DISCUSSION TITLE
Cypress Icon

### DISCUSSION THREAD

**Discussion by jonathanchaffer** [elrumo/macOS_Big_Sur_icons_replacements#1243] [Icon request] [answered]
Hi community,

Hoping to find a Big Sur version of the [Cypress](https://www.cypress.io) icon. Does anyone have one or would be willing to make one? 

![Screen Shot 2021-07-09 at 11 39 34 AM](https://user-images.githubusercontent.com/48372674/125103496-66e0c200-e0aa-11eb-81ee-64b125f29bb2.png)

**Comment by OMGzilla-git** [comment, marked as answer]
Done You can find it in my profile for now under the _Waiting_ section until it gets approved.
https://macosicons.com/u/OMGzilla
